src/components/ec-privacy-policy/ec-privacy-policy.vue

```diff
@@ -31,49 +32,41 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/colors/key-color-scale';
-@import '../../scss/settings/layout';
-@import '../../scss/settings/z-index';
-@import '../../scss/tools/typography';
-@import '../../scss/tools/media-queries';
+<style>
+.ec-privacy-policy {
+  @apply tw-z-level-1;
 
-$ec-privacy-policy-max-width: $body-max-width !default;
+  background: hsla(var(--ec-key-color-level-7), 0.95);
 
-.ec-privacy-policy {
-  z-index: $z-index-level-1;
-  background-color: rgba($level-7, 0.95);
-  position: fixed;
-  bottom: 0;
-  width: 100%;
-  text-align: center;
+  @apply tw-fixed tw-bottom-0;
+  @apply tw-w-full;
+  @apply tw-text-center;
 
   &__content {
-    padding: 24px 48px;
-    max-width: $ec-privacy-policy-max-width;
-    margin: auto;
-    display: flex;
-    flex-wrap: wrap;
-    align-items: center;
-    justify-content: space-between;
+    @apply tw-py-24 tw-px-48;
+
+    max-width: var(--ec-body-max-width);
+
+    @apply tw-m-auto;
+    @apply tw-flex tw-flex-wrap tw-items-center tw-justify-between;
 
     @media screen and (max-width: 640px) {
-      padding: 24px 16px;
+      @apply tw-py-24 tw-px-16;
     }
   }
 
   &__title {
-    @include h4;
+    @apply tw-h4;
 
-    @media screen and (max-width: $ec-privacy-policy-max-width) {
-      display: none;
+    @media screen and (max-width: var(--ec-body-max-width)) {
```

Comment:
I'm pretty sure you cannot use `var()` in the media query and I see that the design is broken because of that. There is a solution for this: `env()` but it is not supported by IE11, doesn't have polyfill, only postcss plugin which requires these values to be resolved during the build. We need to investigate on Monday what to do here. Hardcoding the values is our plan C. Using `env()` without supporting the IE11 is the plan B.

https://caniuse.com/#feat=css-env-function
https://developer.mozilla.org/en-US/docs/Web/CSS/env